README.md

```diff
@@ -1,5 +1,115 @@
-Main branch status: ![Main branch status](https://github.com/epfl-lasa/control_libraries/actions/workflows/build-test.yml/badge.svg) \
-Development branch status: ![Development branch status](https://github.com/epfl-lasa/control_libraries/actions/workflows/build-test.yml/badge.svg?branch=develop)
+<table border="0" width="100%" height="120">
+    <tr>
+        <td width="25%">Branch</td>
+        <td width="75%">Status</td>
+    </tr>
+    <tr>
+        <td width="25%"><a href="https://github.com/epfl-lasa/control_libraries/tree/main">Main</a></td>
+        <td width="75%"><img src="https://github.com/epfl-lasa/control_libraries/actions/workflows/build-test.yml/badge.svg?branch=main"></td>
+    </tr>
+    <tr>
+        <td width="25%"><a href="https://github.com/epfl-lasa/control_libraries/tree/develop">Development</a></td>
+        <td width="75%"><img src="https://github.com/epfl-lasa/control_libraries/actions/workflows/build-test.yml/badge.svg?branch=develop"></td>
+    </tr>
+</table>
+
+# `control_libraries`
+A set of libraries to facilitate the creation of full control loop algorithms,
+including trajectory planning, kinematics, dynamics and control.
+
+## `state_representation`
+
+This library provides a set of classes to represent **states** in **Cartesian** or **joint** spaces.
+The classes define and combine variables such as position, velocity, acceleration and force into
+a consistent internal representation used across the other libraries.
+
+Dependencies: [Eigen3](https://eigen.tuxfamily.org/index.php?title=Main_Page)
+
+[**Documentation**](source/state_representation/README.md)
+
+## `dynamical_systems`
+
+This library provides a collection of classes that behave as differential equations to calculate
+a derivative of a state variable. For example, a position state input might
+yield a desired velocity output. This can be used to generate time-invariant trajectories.
+
+Dependencies: `state_representation`
+
+[**Documentation**](source/dynamical_systems/README.md)
+
+## `robot_model`
+
+This library allows the creation of a robot model from a URDF file and defines many helpful
+rigid-body algorithms for solving kinematic and dynamic problems.
+
+It is a wrapper for [Pinocchio](https://github.com/stack-of-tasks/pinocchio)
+that is compatible with the internal `state_representation` types.
+
+Dependencies: `state_representation`, [Pinocchio](https://github.com/stack-of-tasks/pinocchio)
```

Comment:
maybe here you could link https://stack-of-tasks.github.io/pinocchio/download.html instead of the same link as in the line above

Reply:
valid point